pkgs/tools/misc/ppfetch/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ppfetch";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "yael333";
+    repo = "ppfetch";
+    rev = "acf00175113db7e4bcd04f40596494360d232fdf";
+    sha256 = "sha256-v1l+0MNY8aruoljf9jAzgrpgFiKgqdWTdLv4o78a8kU=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -Dm755 -t $out/bin ppfetch
```

In Nix installed files are usually not writable
```suggestion
    install -Dm555 -t $out/bin ppfetch
```